### DISCUSSION TITLE
記事の画像をアップロードすることができるAPIが欲しい

### DISCUSSION THREAD

**Discussion by TakuKobayashi** [increments/qiita-discussions#513] [要望・アイデア・不具合]
普段、記事執筆していく中でスクリーンショットなどの画像を多く使用しております。
記事を公開する場合には公開前にそれらの画像を手動でアップロードして反映させています。
今回、[qiita-cli](https://github.com/increments/qiita-cli) が公開されました。[qiita-cli](https://github.com/increments/qiita-cli) を経由してローカルで記事を執筆し、Qiitaに投稿するためには、Qiita APIを用いて画像をアップロードできる必要があります。(その後、[qiita-cli](https://github.com/increments/qiita-cli)  の機能追加により [qiita-cli](https://github.com/increments/qiita-cli)  経由で記事を投稿する際に画像も一緒にアップロードすることができるようになります)
そのため画像をアップロードできる Qiita APIの作成をお願いしたいです。

**Comment by ohakutsu** [comment]
@TakuKobayashi 
ご意見ありがとうございます！

私達もQiita CLIで画像のアップロードをできるようにしたいと考えています！
画像アップロード用のAPIをQiita APIとして公開することはまだ断言できませんが、上記の課題解決をできないか検討します！

**Reply by TakuKobayashi** [reply]
@ohakutsu 
お返事ありがとうございます。ぜひこちらご対応いただけることを楽しみにしたいと思います。

**Reply by RickCogley** [reply]
@ohakutsu 如何ですか、CLIでできますでしょうか？

**Reply by ohakutsu** [reply]
@RickCogley
ご質問いただきありがとうございます！

現時点では、CLIでの画像アップロードにはまだ対応できておりません🙏
引き続き、画像アップロードの対応については優先度を上げて検討を進めておりますので、今しばらくお待ちいただけますと幸いです :bow:

**Comment by RickCogley** [comment]
ご検討いただいて、ありがとうございます！

From: ohakutsu ***@***.***>
Date: Wednesday, March 25, 2026 at 14:04
To: increments/qiita-discussions ***@***.***>
Cc: Rick Cogley ***@***.***>, Mention ***@***.***>
Subject: Re: [increments/qiita-discussions] 記事の画像をアップロードすることができるAPIが欲しい (Discussion #513)


@RickCogley<https://github.com/RickCogley>
ご質問いただきありがとうございます！

現時点では、CLIでの画像アップロードにはまだ対応できておりません🙏
引き続き、画像アップロードの対応については優先度を上げて検討を進めておりますので、今しばらくお待ちいただけますと幸いです 🙇
